src/commands/serve.ts

```diff
@@ -28,6 +28,15 @@ export class ServeCommand implements Command {
       browser: options.browser,
       componentDir: options['component-dir'],
       packageName: options['package-name'],
+    };
+
+    if (options.env) {
+      console.log(options.env.impl);
+      if (!options.env.impl.serve) {
+        throw new Error(`Serve unimplemented in environment ${options.env.name}`);
```

Comment:
I'd rather each environment be able to implement as few hooks as possible and work with the default build/project/etc whenever possible.